extensions/standard-processors/processors/LogAttribute.cpp

```diff
@@ -88,23 +88,14 @@ void LogAttribute::initialize() {
 }
 
 void LogAttribute::onSchedule(const std::shared_ptr<core::ProcessContext> &context, const std::shared_ptr<core::ProcessSessionFactory> &factory) {
-  core::Property flowsToLog = FlowFilesToLog;
-
-  if (getProperty(FlowFilesToLog.getName(), flowsToLog)) {
-    // we are going this route since to avoid breaking backwards compatibility the get property function doesn't perform validation ( That's done
-    // in configuration. In future releases we can add that exception handling there.
-    if (!flowsToLog.getValue().validate("Validating FlowFilesToLog").valid())
-      throw Exception(PROCESS_SCHEDULE_EXCEPTION, "Invalid value for flowfiles to log: " + flowsToLog.getValue().to_string());
-    flowfiles_to_log_ = flowsToLog.getValue();
-  }
```

Comment:
This line, if I wrote like: 
```
if (!context->getProperty(FlowFilesToLog.getName(), flowfiles_to_log_)) {
      throw ...
}
```
Then the GetTCPTest (only this) will fail due to exception, so I guess that test's configuration has issues, will take a look later.

Reply:
You are right. 

Initialise() is not called in the testcases, which would set the default value. 
Added that call to every testcase fixes the issue.